### ISSUE TITLE
getsentry/sentry-laravel

### ISSUE THREAD

**Issue by AidasK** [jenssegers/laravel-raven#23]
Should this library be replaced by official one https://github.com/getsentry/sentry-laravel?

**Comment by apollonin** [comment]
No, I think this repo gives ability to Log any message like Log::debug(), Log::info(), Log::warning() and so on to Sentry. 
But official one give only ability to track Exceptions.
